docs/stacks-guide-azure.md

```diff
@@ -0,0 +1,429 @@
+cluster-scoped via `--cluster` flag.
+```
+kubectl crossplane stack generate-install --cluster 'crossplane/stack-azure:master' stack-azure | kubectl apply --namespace crossplane-system -f -
+```
+
+We have installed the Azure stack into the `crossplane-system` namespace, but we
+want to group our Azure-specific resources in their own environment namespaces.
+For the purpose of this guide, we will create all Azure-specific resources in
+the `azure-infra-dev` namespace, and all application-specific resources in the
+`app-project1-dev` namespace. Let's create these namespaces before we get
+started:
+
+```
+kubectl create namespace azure-infra-dev
+kubectl create namespace app-project1-dev
+```
+
+## Configure Azure Account
+
+We will make use of the following services on Azure:
+
+*   Resource Group
+*   AKS
+*   Azure Database for MySQL
+*   Virtual Network
+*   Subnetwork
+*   Virtual Network Rule
+
+In order to utilize each of these services, you will need to follow the Adding
+Microsoft Azure to Crossplane [guide](cloud-providers/azure/azure-provider.md)
+to obtain appropriate credentials in a JSON file referred to as
+`crossplane-azure-provider-key.json`.
+
+## Configure Crossplane Azure Provider 
+
+Before creating any resources, we need to create and configure a cloud provider
+in Crossplane. This helps Crossplane know how to connect to the cloud provider.
+All the requests from Crossplane to Azure can use that resource as their
+credentials. The following command assumes that you have a
+`crossplane-azure-provider-key.json` file that belongs to the account you’d like
+Crossplane to use.
+
+```
+export BASE64ENCODED_AZURE_PROVIDER_CREDS=$(base64 crossplane-azure-provider-key.json | tr -d "\n")
+```
+
+Now we’ll create our `Secret` that contains the credential and `Provider`
+resource that refers to that secret:
+
+```
+cat > provider.yaml <<EOF
+---
+# Azure Admin service account secret - used by Azure Provider
+apiVersion: v1
+kind: Secret
+metadata:
+  name: demo-provider-azure-dev
+  namespace: azure-infra-dev
+type: Opaque
+data:
+  credentials: $BASE64ENCODED_AZURE_PROVIDER_CREDS
+---
+# Azure Provider with service account secret reference - used to provision resources
+apiVersion: azure.crossplane.io/v1alpha2
+kind: Provider
+metadata:
+  name: demo-azure
+  namespace: azure-infra-dev
+spec:
+  credentialsSecretRef:
+    name: demo-provider-azure-dev
+    key: credentials
+EOF
+
+kubectl apply -f provider.yaml
+```
+
+The name of the `Provider` resource in the file above is `demo-azure`; we'll use
+the name `demo-azure` to refer to this provider when we configure and set up
+other Crossplane resources.
+
+We also will need to use our Azure subscription id to provision some of the
+resources we will need. You can set an environment variable so that you will
+have access when creating resources that require it:
+
+```bash
+export SUBSCRIPTION_ID=$(cat crossplane-azure-provider-key.json | jq -j '.subscriptionId')
```

Comment:
I believe this is the first time `jq` is mentioned; I am thinking we should add a prerequisites section that calls it out as a dependency, or try to do this in a way that doesn't use `jq`.

Reply:
I am going to be testing this guide today and will see if I can implement a workaround for this and the later use of `jq`